### DISCUSSION TITLE
Trying to use in a Visual Studio Extension Tool Window - Unable to load file or assembly CefSharp.Wpf

### DISCUSSION THREAD

**Discussion by ricardorjpires** [cefsharp/CefSharp#4697] [Q&A]
I have been working in a Extension for Visual Studio using CefSharp.Wpf and I can't resolve this problem with the assemblies.
I already try a lot of things and none worked.
When I call Cef.Initialize the assembly error changes to CefSharp.Core.Runtime

It's possible to use this library in a Visual Studio Extension Project (tool window)?

VS2022
.net Framework 4.8
Visual Studio Extension Project
CefSharp.Wpf 120

System.Windows.Markup.XamlParseException: 'Unable to load file or assembly 'CefSharp.Wpf, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The system cannot find the specified file.'

**Comment by amaitland** [comment]
> It's possible to use this library in a Visual Studio Extension Project (tool window)?

For plugins/extensions you need to load the browser in a separate process.

There's an example of running CefSharp out of process at https://github.com/cefsharp/CefSharp.OutOfProcess
Alternative is to switch to a different browser that by default runs the browser in a separate process.

**Reply by ricardojosezup** [reply]
Thank you! I'll try.
